029-IoTEdge/Student/Challenge-07.md

```diff
@@ -0,0 +1,116 @@
+# Challenge 7: Connect to Devices with Device Streams
+
+[< Previous Challenge](./Challenge-06.md) - **[Home](../README.md)**
+
+## Pre-requisites
```

Comment:
Guys... these sorts of pre-requisites that involve installing stuff need to go in a "Challenge 0: Ready, Set, GO!" type of challenge in the student guide that is done first (or pre-event).

Please see 001-IntroToKubernetes for an example.

Reply:
Added to the challenge-1 items needed for success.  Let me know what you think

Reply:
Looks great BUT you left the empty header here, just delete it and we're good.